client/landing/gutenboarding/available-designs.ts

```diff
@@ -34,9 +35,17 @@ export const getDesignImageUrl = ( design: Design ) => {
 		{
 			font_headings: design.fonts.headings,
 			font_base: design.fonts.base,
+			viewport_height: 700,
 		}
 	);
-	return mshotsUrl + encodeURIComponent( previewUrl );
+	const mshotsRequest = addQueryArgs( mshotsUrl + encodeURIComponent( previewUrl ), {
+		vpw: 1200,
+		vph: 3072,
```

Comment:
Sadly this is capped at `1200`:

https://github.com/Automattic/mShots/blob/100faf8c8c1148fd8584713999ab944e6fcf24a3/public_html/class-mshots.php#L15

Reply:
@simison: Seems to work ok?
https://s.wordpress.com/mshots/v1/https%3A%2F%2Fgithub.com%2FAutomattic%2Fwp-calypso%2Fpull%2F43428?vph=3072
vs
https://s.wordpress.com/mshots/v1/https%3A%2F%2Fgithub.com%2FAutomattic%2Fwp-calypso%2Fpull%2F43428?vph=1200

Reply:
Yep, looks like the version we have running supports greater max width/height (p3topS-Ia-p2#comment-2967)

Reply:
Oh, that's wonderful! I had missed an update in that P2. (_How could I!_)